bindings/cpp/include/wtf/macros.h

```diff
@@ -44,6 +45,19 @@
 #define WTF_THREAD_ENABLE(name) \
   WTF_THREAD_ENABLE_IF(kWtfEnabledForNamespace, name)
 
+// Enables WTF tracing for any thread based on whether the current namespace
+// is enabled. Thread name will be auto-generated.
+#define WTF_ALL_THREADS_ENABLE() \
```

Comment:
Also, feel free to kick this back to me if you don't have time/energy to update your change with my comments and I can take care of it. Thanks!